packages/core/nuxt-theme-module/theme/components/LoginModal.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <SfModal
-    :visible="isLoginModalOpen"
+    :visible="true"
```

Comment:
I think this prop should default to `true`. WDYT @AdamPawlinski and @justyna-13 ?